core/nodejsActionBase/bin/compile

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
```

Comment:
I'm curious why there's so much logic in the `/bin/compile`.   In our previous discussions, I came to thing that the go proxy would be there to take care of log forwarding but we would not encouraging full action loop semantics for end users.    Is it because it's a substitute for things that used to be done in a special way in the deleted `app.js` or elsewhere?

Reply:
Pretty much, yes. It's doing almost 1:1 what `app.js` would've done before (I wanted to reuse it but that created more issues down the line). It looks much more verbose though because of the filesystem operations.